reana_server/utils.py

```diff
@@ -63,27 +63,27 @@ def get_user_from_token(token):
 def _get_users(_id, email, user_token, token):
```

Comment:
maybe we can rename `token` to `admin_token`, what do you think? Also in `_create_user(..)`

Reply:
Oops, should've changed it as in https://github.com/reanahub/reana-server/pull/64/files#diff-aa882553e3d2579ee378587080f3db34R133, I went with `admin_access_token` and `user_access_token` WDYT @diegodelemos ?

Reply:
even better 🍎